### PR TITLE
Test pull request from github

### PR DESCRIPTION
**Pull request by inode9**

### CODE REVIEW

File: 3.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+3 from inode9
```
Comment (maciejszewczyk) [inline 3.txt:1]: Please replace 3 with 3.txt

File: 4.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+4 from inode9
```
Comment (maciejszewczyk) [inline 4.txt:1]: Start a review

File: 3.txt (hunk 1 above)
Comment (inode9) [reply 3.txt:1]: OK

File: 4.txt (hunk 2 above)
Comment (maciejszewczyk) [reply 4.txt:1]: bbb

**Review by inode9 — COMMENTED** [review]
fixed

**Review by maciejszewczyk — APPROVED** [review]

**Pull request merged**